src/test/java/core/basesyntax/dao/StorageDaoImplTest.java

```diff
@@ -0,0 +1,79 @@
+package core.basesyntax.dao;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import core.basesyntax.model.FruitModel;
+import core.basesyntax.storage.FruitStorage;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class StorageDaoImplTest {
+    private StorageDaoImpl storageDao;
+
+    @Before
+    public void setUp() throws Exception {
+        storageDao = new StorageDaoImpl();
+        FruitStorage.fruitStorage.put("apple", 10);
+    }
+
+    @After
+    public void tearDown() throws Exception {
+        FruitStorage.fruitStorage.clear();
+    }
+
+    @Test(expected = RuntimeException.class)
+    public void getAmount_getAbsentKey_Exception() {
```

Comment:
```suggestion
    public void getAmount_getAbsentKey_Exception() {
```
In my opinion, you should rename all methods in format 'methodName_Ok(notOk) but it's optional, because you set your own names instead 'Ok' and 'notOk', which are enough informative.